src/constants/systemGeos.ts

```diff
@@ -1,6 +1,9 @@
+export const SYSTEM_INTERNATIONAL_CODE = "xab";
+export const SYSTEM_INTERNATIONAL_NAME = "international";
+
 export const systemGeoCodes = [
   "xaa", // No Geography
```

Comment:
Why isn't this a const too, if we're pulling XAB out into a const?

Reply:
Should be, I was being lazy